tests/docs/smoke-all/2024/05/03/9548.qmd

```diff
@@ -6,12 +6,14 @@ _quarto:
   tests:
     revealjs:
       ensureHtmlElements:
-        - ['head > link[rel="stylesheet"][href$="quarto-4f64e0fc78c89fc90127583bb01c366c.css"]']
+        - []
         - ['head > link[rel="stylesheet"][href$="beige.css"]']
+      ensureFileRegexMatches:
+        - ['<link .* href=.+revealjs/dist/theme/quarto-([a-f0-9]{32})\.css']
 ---
 
 # Revealjs theme handling
 
-User provided theme should be used to build a `quarto-<hash>.css` using SASS and the `theme: beige` should internally by overridden to `theme: quarto` so that the later is added in Pandoc's template
+User provided them9548_files/libs/revealjs/dist/theme/e should be used to build a `quarto-<hash>.css` using SASS and the `theme: beige` should internally by overridden to `theme: quarto` so that the later is added in Pandoc's template
```

Comment:
> `them9548_files/libs/revealjs/dist/theme/e`

What happened here? (Not that it matters, but it looks weird.)

Reply:
Oh yes definitely weird. How did I missed that ! Thank you. 
Wrong copy page in the wrong place 😭 Thank you

Reply:
Fixed